LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs

```diff
@@ -417,7 +418,8 @@ public void Update()
 
         for (int i = 0; i < Voltages.Length; i++)
         {
-            if (Chip != Chip.NCT6687D)
+            // if (Chip != Chip.NCT6687D)
+            if (Chip is not Chip.NCT6687D and Chip.NCT6686D)
```

Comment:
You missed a "not" here I think.

Reply:
and rebase.

Reply:
Awesome. I changed it on my files. New to Github, how do i commit it to this?

Reply:
With GitHub Desktop you can rebase (Branch menu item, somewhere at the bottom).

Reply:
Awesome. I think i did it. :)

Reply:
![image](https://user-images.githubusercontent.com/11328522/201101051-30aed337-7f52-46d0-bbc6-f877d69b42f8.png)
Seeing more voltage items.
Would need help here, as I'm unsure about them.

Reply:
These values look odd, but I'm not sure this chipset is nct6686d, it should have id 0xd440.

Reply:
I got the ID `0xD441` off HWinfo's export.

Reply:
I'm not sure that's correctly identified by hwinfo then, it should be 0xd440: https://github.com/torvalds/linux/blob/master/drivers/hwmon/nct6683.c#L72
Which might explain the odd values you're seeing.

Reply:
Just tried 0xd440, but the first 9 values are still not showing.